src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DwpState.java

```diff
@@ -41,6 +41,7 @@ public enum DwpState {
     SET_ASIDE_REQUESTED("setAsideRequested", "Set Aside Requested"),
     STATEMENT_OF_REASONS_GRANTED("statementOfReasonsGranted", "SOR Granted"),
     STATEMENT_OF_REASONS_REFUSED("statementOfReasonsRefused", "SOR Refused"),
+    STATEMENT_OF_REASONS_REQUESTED("statementOfReasonsRequested", "SOR requested"),
```

Comment:
Maybe capitalise the R to be consistent with the others

Reply:
done